netci.groovy

```diff
@@ -5,6 +5,12 @@ import jobs.generation.*;
 def project = GithubProject
 def branch = GithubBranchName
 
+// 'isPullR' used as a parameter elsewhere in the file
+def isPullR = false
```

Comment:
I am asking why you are defining this at all. You should need to define nothing.

Just modify `static void addVsiMultiScm(def myJob, def project) {` to be `static void addVsiMultiScm(def myJob, def project, def isPR) {` [Line 198](https://github.com/dotnet/roslyn-project-system/pull/1512/files#diff-2d212ad84506951c8f928e21aed24580L198)

and modify `addVsiMultiScm(newVsiJob, project)` to be `addVsiMultiScm(newVsiJob, project, isPR)` [Line 167](https://github.com/dotnet/roslyn-project-system/pull/1512/files#diff-2d212ad84506951c8f928e21aed24580L167)